backend/de.metas.acct.base/src/main/sql/postgresql/ddl/views/tax_accounts_details_v.sql

```diff
@@ -97,9 +97,9 @@ FROM (SELECT fa.vatcode                        AS vatcode,
              fa.ad_org_id,
              fa.ad_client_id
       FROM fact_acct fa
-               LEFT OUTER JOIN c_elementvalue ev ON ev.c_elementvalue_id = fa.account_id
-               LEFT OUTER JOIN c_tax tax ON fa.c_tax_id = tax.c_tax_id
-               LEFT OUTER JOIN C_Currency c ON fa.c_Currency_ID = c.C_Currency_ID
+               INNER JOIN c_elementvalue ev ON ev.c_elementvalue_id = fa.account_id
+               INNER JOIN c_tax tax ON fa.c_tax_id = tax.c_tax_id
```

Comment:
hmmm, before was outer join.
wondering how it was working before when Fact_Acct.C_Tax_ID was not set?

Because now we are just excluding those records... which I am not so happy about it.... not sure.

pls clarify the impact.